gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/ApiDuplicatorServiceImpl.java

```diff
@@ -329,10 +329,20 @@ private UpdateApiEntity convertToEntity(final ExecutionContext executionContext,
 
         //create group if not exist & replace groupName by groupId
         if (importedApi.getGroups() != null) {
-            Set<String> groupNames = new HashSet<>(importedApi.getGroups());
+            Set<String> groupIdOrNames = new HashSet<>(importedApi.getGroups());
             importedApi.getGroups().clear();
-            for (String name : groupNames) {
-                List<GroupEntity> groupEntities = groupService.findByName(executionContext.getEnvironmentId(), name);
+            for (String name : groupIdOrNames) {
```

Comment:
I think most of user of the UI import are now used to this behaviour. Should we wrap this in a conditional so that it only concern kubernetes API ?